src/infra/marketplace/assets/drupal.json

```diff
@@ -50,6 +50,16 @@
     "chown -h nobody:nogroup %installDirectory%",
     "chown -R nobody:nogroup -L /app/drupal-%installHostname%-%installUuid%/"
   ],
+  "uninstallCmdSteps": [
+    "os db delete-user -n drupalDb_%installUuid% -t mysql -u drupalUser_%installUuid%",
+    "os db delete -n drupalDb_%installUuid% -t mysql",
+    "mkdir -p /app/.trash/drupal-%installHostname%-%installUuid%",
+    "find %installDirectory% -mindepth 1 -maxdepth 1 -exec mv -t /app/.trash/drupal-%installHostname%-%installUuid% {} +",
+    "rm -rf /app/drupal-%installHostname%-%installUuid%",
+    "rm -rf %installDirectory%",
+    "mkdir -p %installDirectory%",
+    "find /app/.trash/drupal-%installHostname%-%installUuid% ! \\( -name \".editorconfig\" -o -name \".gitattributes\" -o -name \"composer.json\" -o -name \"composer.lock\" -o -name \"vendor\" -o -name \"web\" \\) -maxdepth 1 -exec mv -t %installDirectory% {} +"
```

Comment:
I'm wondering... the logic here is to move the files not matching Drupal files back to the installDirectory. Wouldn't that be possible to achieve if you only moved the files belonging to Drupal to the .trash dir and then have on the code a check to see if installDirectory is a symlink and then apply the corrections? I mean, the idea of the exception here is to convert the symlink back a regular directory but that could be checked on the code itself so you don't need these steps here and on Laravel, right?

Reply:
With this new implementation, I was able to review the commands being executed and improve them to ensure that some previous problems were not carried over to this new version.